torch/csrc/jit/python/pybind_utils.h

```diff
@@ -60,10 +68,22 @@ struct PythonFutureWrapper {
       auto output = graph->insert(aten::wait, {fut_val});
       jit::tracer::setValueTrace(fut->value(), output);
     }
-    return fut->value();
+    py::object py_obj;
```

Comment:
why this needs to be out of the GIL? looks like we can do the following?

```c++
auo py_obj = toPyObject(fut->value());
```

Reply:
@mrshenli There is a `fut->wait();` above to return IValue, we should not hold GIL waiting it.

Reply:
I mean the GIL below this line. Sth like the following. But this is just a nitpick. Feel free to ignore. :)

```c++
    // py::object py_obj;
    {
      // acquiring GIL as toPyObject creates new py::object
      // without grabbing the GIL.
      py::gil_scoped_acquire acquire;
      auto py_obj = toPyObject(fut->value());
      if (unwrap_func) {
        (*unwrap_func)(py_obj);
      }
      return py_obj;
    }
```

Reply:
Updated.